modules/cli/src/main/scala/scala/cli/exportCmd/Mill.scala

```diff
@@ -46,6 +46,11 @@ final case class Mill(
       MillProject(scalaVersion = Some(sv))
   }
 
+  private def scalaCompilerPlugins(buildOptions: BuildOptions): MillProject =
+    MillProject(scalaCompilerPlugins =
+      buildOptions.scalaOptions.compilerPlugins.toSeq.map(_.value.render)
```

Comment:
@carlosedp  Thanks for your contribution, it seems that the `.reder` method returns the wrong number of colons for the compiler plugin.

Therefore `mill` fail with downloading plugin dependency:
```
newproject.enablePluginScalacOptions 
Resolution failed for 1 modules:--------------------------------------------
  com.olegpy:better-monadic-for_3:0.3.1 
	not found: /home/runner/.ivy2/local/com.olegpy/better-monadic-for_3/0.3.1/ivys/ivy.xml
	not found: https://repo1.maven.org/maven2/com/olegpy/better-monadic-for_3/0.3.1/better-monadic-for_3-0.3.1.pom
```

Reply:
Thanks for picking that up @lwronski! I haven't noticed that this was the problem.
Fixed on latest push.

Reply:
@carlosedp  It wasn't a problem, sorry for pointing the problem wrong.

The problem is, that `ExportMillTestDefinitions` tests are running with every version of Scala - [2.12, 2.13, 3]. But `better-monadic-for` is published only for `2.12` and `2.13`. Therefore the test which used Scala 3 fail in CI, because mill can not download `better-monadic-for` for Scala 3.

You can run Mill export tests with Scala 3 in this way:
```
 ./mill integration.test.jvm 'scala.cli.integration.ExportMillTestsDefault.JVM custom project name'
```

I think you should revert the last commit because the `render` works fine, but this PR needs to update tests. Maybe it would be better to add separate tests and test only export scala compiler plugins with specific scala versions. It could be added small tests in `ExportMillTestsDefault` and in the test you can hardcode the scala version, for example, 3.1.0.

Reply:
Got it... Thanks a lot for the tips.
I'm figuring out a way to test it locally since I'm developing on a Mac and these tests only run on Linux.

Reply:
I think, you can temporary change definition of `ExportMillTestDefinitions.runExportTests` to:
```
  protected def runExportTests: Boolean = true
```
to local development. And before pushing the commit back to the previous implementation of `ExportMillTestDefinitions.runExportTests`.

Maybe, scala-cli should turn on ExportMillTest to run on Mac CI. I will investigate it.

Reply:
Nice, setting it to true worked (dunno why it didn't work the last time I tried). I filtered my test with `if (runExportTests && !actualScalaVersion.startsWith("3."))`.

I'm running it as `./mill integration.test.jvm 'scala.cli.integration.ExportMillTestsDefault.*'` but it's not running the tests for Scala 2... is there a way to pass the scala version as an argument?

```
scala.cli.integration.ExportMillTestsDefault:
  + JVM 22.97s
  + JVM custom project name 19.303s
  + Scala.js 20.15s
```

Reply:
If you want to run test for Scala 2 you can do it in this way:
```
./mill integration.test.jvm 'scala.cli.integration.ExportMillTests213.*' 
./mill integration.test.jvm 'scala.cli.integration.ExportMillTests212.*' 
```

Eg `ExportMillTests212` runs tests that are defined in `ExportMillTestDefinitions` using Scala 2.12. And ExportMillTestsDefault uses Scala 3.

Reply:
Oh, got it... my bad I didn't see it before!
Running right now and will push in a couple minutes.

Reply:
Yay, worked!

```
scala.cli.integration.ExportMillTests213:
  + JVM 28.302s
  + JVM custom project name 24.978s
  + JVM with compiler plugin 26.33s
  + Scala.js 25.377s
  + Scala Native 39.551s
  + scalac options 20.64s
  + pure java 9.633s
  + custom JAR 21.0s
```